tests/docker-3par-cho/README.md

```diff
@@ -0,0 +1,43 @@
+## HPE 3PAR Docker Volume Plugin- Continuous Hours (CHO) Testing
+
+### Overview
+This document details the steps necessary to perform a Reliability testing or we can say Continuous Hours of Operation (aka CHO) tests against HPE 3PAR Docker Volume Plugin. This testing allows us to verify that no bugs creep up after lengthy use of those components of volume plugin.
+
+### Setup
+Setup Docker Engine environment normally.
+
+Install Docker SDK using "pip install docker" command.
+
+Install and enable the HPE 3PAR Docker Volume plugin
+
+```
+Pull tests from Git
+git clone https://github.com/sumitshatwara/docker-3par-cho
```

Comment:
This need a change.

Reply:
Correct repository will be 
`git clone https://github.com/hpe-storage/python-hpedockerplugin -b test_automation`